crates/oxc_linter/src/rules/react/jsx_curly_brace_presence.rs

```diff
@@ -1014,61 +1095,61 @@ fn test() {
         ),
         (
             "
-			        <MyComponent>
-			          {'%'}
-			        </MyComponent>
-			      ",
-            "
-			        <MyComponent>
-			          %
-			        </MyComponent>
-			      ",
-            Some(json!([{ "children": "never" }])),
-        ),
-        (
-            "
-			        <MyComponent>
-			          {'foo'}
-			          <div>
-			            {'bar'}
-			          </div>
-			          {'baz'}
-			        </MyComponent>
-			      ",
+        	        <MyComponent>
+        	          {'%'}
+        	        </MyComponent>
+        	      ",
             "
-			        <MyComponent>
-			          foo
-			          <div>
-			            bar
-			          </div>
-			          baz
-			        </MyComponent>
-			      ",
+        	        <MyComponent>
+        	          %
+        	        </MyComponent>
+        	      ",
             Some(json!([{ "children": "never" }])),
         ),
         (
             "
-			        <MyComponent>
-			          {'foo'}
-			          <div>
-			            {'bar'}
-			          </div>
-			          {'baz'}
-			          {'some-complicated-exp'}
-			        </MyComponent>
-			      ",
+        	        <MyComponent>
+        	          {'foo'}
+        	          <div>
+        	            {'bar'}
+        	          </div>
+        	          {'baz'}
+        	        </MyComponent>
+        	      ",
             "
-			        <MyComponent>
-			          foo
-			          <div>
-			            bar
-			          </div>
-			          {'baz'}
-			          {'some-complicated-exp'}
-			        </MyComponent>
-			      ",
+        	        <MyComponent>
+        	          foo
+        	          <div>
+        	            bar
+        	          </div>
+        	          baz
+        	        </MyComponent>
+        	      ",
             Some(json!([{ "children": "never" }])),
         ),
+        // (
+        //     "
+        // 	        <MyComponent>
+        // 	          {'foo'}
+        // 	          <div>
+        // 	            {'bar'}
+        // 	          </div>
+        // 	          {'baz'}
+        // 	          {'some-complicated-exp'}
+        // 	        </MyComponent>
+        // 	      ",
+        //     "
+        // 	        <MyComponent>
+        // 	          foo
+        // 	          <div>
+        // 	            bar
+        // 	          </div>
+        // 	          {'baz'}
+        // 	          {'some-complicated-exp'}
```

Comment:
Here is a bug, that this will be reported (and fixed). But I guess it has something to do with line breaks / spaces?